### DISCUSSION TITLE
Πρόσθετη συνάντηση

### DISCUSSION THREAD

**Discussion by epidrome** [upatras-hci/iv#79] [Polls]
Αυτήν την εβδομάδα έχω μεγάλο φόρτο οπότε δεν είμαι διαθέσιμος και σήμερα φαίνεται πως δεν βόλευε κάποιους, ας κάνουμε λοιπόν μια τελευταία προσπάθεια για μια κοινά αποδεκτή ημερομηνία την επόμενη εβδομάδα

**Comment by e-liouta** [comment]
Εγώ επειδή έχω παρουσιάσει και από ότι κατάλαβα το βάρος είναι στα παιδιά που δεν πρόλαβαν να δείξουν κάποια πρόοδο δεν ξέρω κατά πόσο είναι σωστό να ψηφίσω. Ωστόσο θα ήθελα να συμμετέχω και αν γίνει τη Δευτέρα 13 Φεβ. 7μμ θα μπω να υποστηρίξω τους συναδέλφους μου και να συμμετέχω στον σχολιασμό. Στόχος μου είναι να έχω κάνει μέχρι τότε το pull request γιατί έπειτα θα λείπω ταξίδι.

**Comment by e-liouta** [comment]
Τελικά θα γίνει η συνάντηση σήμερα στις 7μμ;

**Reply by epidrome** [reply]
κανείς δεν έχει κάνει γραπτή δήλωση παρουσίασης #74 

εγώ ευχαρίστως να μπω στις 7μμ, αλλά αν δεν γίνει κάποια από τις εκκρεμείς παρουσιάσεις για σχολιασμό δεν θα το κρατήσουμε πολύ

**Reply by e-liouta** [reply]
Ναι επειδή μιλάμε με τα παιδιά πρόκειται να παρουσιάσουν οι περισσότεροι τουλάχιστον που έχω επικοινωνήσει απλά από κεκτημένη ταχύτητα απαντήσαμε εδώ και δεν σκεφτήκαμε να συμπληρώσει κάποιος το  [74](https://github.com/upatras-hci/iv/discussions/74) οπότε τα λέμε στις 7!
